lib/git/printable.go

```diff
@@ -0,0 +1,40 @@
+package git
```

Comment:
It depends on the `git` package though as it makes the call to git functions. Yes this file makes data printable, but it is also only doing so for the results of git functions. Putting it in its own package means that package will depend on the git folder or we if we don't want that dependency, we would need more generic functions that get passed in data.